src/raw/dockerapi/fetcher.go

```diff
@@ -57,8 +58,27 @@ func (f *Fetcher) networkMetrics(containerStats types.StatsJSON) raw.Network {
 	return aggregatedMetrics
 }
 
-func (f *Fetcher) cpuMetrics(containerStats types.StatsJSON) raw.CPU {
-	return raw.CPU{}
+func (f *Fetcher) cpuMetrics(container types.ContainerJSON, containerStats types.StatsJSON) raw.CPU {
+	cpuStats := containerStats.CPUStats
```

Comment:
perhaps just passing this as func parameter?

Reply:
Changed here: [6c61dc8](https://github.com/newrelic/nri-docker/pull/214/commits/6c61dc89c32b737e0dbd8c3aec4ea60863883442)